bftclient/test/bft_client_api_tests.cpp

```diff
@@ -271,7 +271,8 @@ class RetryBehavior {
     not_heard_from_yet.erase(msg.destination);
     if (not_heard_from_yet.empty()) {
       auto reply = replyFromRequest(msg);
-      client_receiver->onNewMessage((NodeNum)msg.destination.val, (const char*)reply.data(), reply.size());
+      client_receiver->onNewMessage(
+          (NodeNum)msg.destination.val, (const char*)reply.data(), reply.size(), (NodeNum)msg.destination.val);
```

Comment:
No need to pass the same value twice - let the default value take place. Everywhere in this file.

Reply:
updated.